flash/core/data/transforms.py

```diff
@@ -75,9 +78,13 @@ def forward(self, inputs: Any):
         result = list(inputs) if isinstance(inputs, Sequence) else [inputs]
         for transform in self.children():
             inputs = result
+            if hasattr(transform, "_params") and bool(transform._params):
```

Comment:
why the bool check is needed ?

Reply:
Not sure, I copied that from your code haha

Reply:
then find. I'll check later why is really needed since I don't recall the reason why this was added